common/kernel/timing_log.cc

```diff
@@ -168,7 +174,27 @@ static void log_crit_paths(const Context *ctx, TimingResult &result)
         log_info("Critical path report for cross-domain path '%s' -> '%s':\n", start.c_str(), end.c_str());
         print_path_report(report);
     }
-};
+
+    // Min delay violated paths
+    // Show maximum of 10
```

Comment:
Hm, compared to max delay violations where we display one per clock (pair) this seems like a behaviour change. If we are going to show a maximum of 10, we should probably be sorting to show the worst 10, which I might have missed, but I can't see anywhere.

Reply:
This is true. I also thought to show the worst per clock pair the same way we do for max delay. The other question remaining is whether we should add the hold slack to slack histogram.